docs/sources/panels/expressions.md

```diff
@@ -0,0 +1,189 @@
++++
+title = "Expressions"
+weight = 800
++++
+
+# Server-side expressions
+
+> **Note:** This documentation is for a beta feature.
+
+Server-side expressions allow you to manipulate data returned from queries with math and other operations. Expressions create new data and do not manipulate the data returned by data sources, aside from some minor data restructuring to make the data acceptable input for expressions.
+
+## Using expressions
+
+The primary use case for expressions is with alerting. Like alerting, processing is done server-side, so expressions can operate without a browser session. However, expressions can be used with backend data sources and visualization as well.
+
+\\ KYLE - IIRC, expressions do not work with current alerting. Should we even mention alerting at all? If we do, then we need to make clear which alerting we are talking about. We can either talk about AlertingNG here, or we can remove references to alerting and update this topic when we release AlertingNG.
```

Comment:
Correct that expressions do not work with current alerting. So yes sohuld change this to next gen alerting.

Reply:
Will that be the official name in the UI? 

It might be easier to remove the AlertingNG content for now and add it in later. What do you think?